mne/viz/_brain/_brain.py

```diff
@@ -2723,7 +2805,8 @@ def screenshot(self, mode='rgb', time_viewer=False):
         screenshot : array
             Image pixel values.
         """
-        img = self._renderer.screenshot(mode)
+        with self._ensure_screenshot_size():
+            img = self._renderer.screenshot(mode)
```

Comment:
This feels like a hack. `brain.screenshot` should not be something that changes the window size. The window should be sized properly at the beginning like it is on `master`.

Reply:
It is one :sweat: It's all I came up after investing a day into this. I did fd03c18 to move on.

Reply:
The problem is that the left area is too big, so it forces the size to be too large. I'm going to try putting it inside a scrollable area. This is how freeview works. Also we need it for sustainability anyway as we add more controls.

Reply:
![Screenshot from 2021-02-26 10-14-57](https://user-images.githubusercontent.com/2365790/109318157-8114b480-781b-11eb-9810-bccd6adf4873.png)

Reply:
All better:

![Screenshot from 2021-02-26 13-41-28](https://user-images.githubusercontent.com/2365790/109341395-5be26f00-7838-11eb-8cc0-5a559f29d7f6.png)

will push a commit shortly